publication-commons/src/test/resources/features/AccessedToPublishedMaterial.feature

```diff
@@ -0,0 +1,64 @@
+Feature:
+  1. DynamoDBPublicationService allows a user to read any document that is marked as published and
+  therefore it is publicly available.
+
+  2. DynamoDBPublicationService does not allow any user to modify or delete any document that is marked
+  as published.
+
+  Notes:
+  1. By "published" we mean that a document has been published in a Journal, Conference proceedings,
```

Comment:
Published in the sense of NVA means that it has status PUBLISHED, there is no requirement that it has simultaneously been published elsewhere — the data may be so-called "grey literature"